### ISSUE TITLE
Play Store reporting Medic app crashes

### ISSUE THREAD

**Issue by garethbowen** [medic/cht-android#338]
**Describe the bug**
Play Store is reporting the "Medic" android app is crashing for 15% of users many with a `android.app.Fragment$InstantiationException`. The Medic app isn't "production" but we should investigate to ensure this doesn't impact anyone else, and prioritise accordingly.

**To Reproduce**
Not sure how to reproduce it.

**Expected behavior**
No crashes.

**Logs**
```
Exception java.lang.RuntimeException: Unable to start activity ComponentInfo{org.medicmobile.webapp.mobile/org.medicmobile.webapp.mobile.EmbeddedBrowserActivity}: android.app.Fragment$InstantiationException: Unable to instantiate fragment t.v: could not find Fragment constructor
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3491)
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:3643)
  at android.app.ActivityThread.handleRelaunchActivityInner (ActivityThread.java:5505)
  at android.app.ActivityThread.handleRelaunchActivity (ActivityThread.java:5404)
  at android.app.servertransaction.ActivityRelaunchItem.execute (ActivityRelaunchItem.java:69)
  at android.app.servertransaction.TransactionExecutor.executeCallbacks (TransactionExecutor.java:135)
  at android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:95)
  at android.app.ClientTransactionHandler.executeTransaction (ClientTransactionHandler.java:58)
  at android.app.ActivityThread.handleRelaunchActivityLocally (ActivityThread.java:5464)
  at android.app.ActivityThread.access$3400 (ActivityThread.java:244)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2115)
  at android.os.Handler.dispatchMessage (Handler.java:106)
  at android.os.Looper.loop (Looper.java:223)
  at android.app.ActivityThread.main (ActivityThread.java:7707)
  at java.lang.reflect.Method.invoke
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:612)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:997)
Caused by android.app.Fragment$InstantiationException: Unable to instantiate fragment t.v: could not find Fragment constructor
  at android.app.Fragment.instantiate (Fragment.java:562)
  at android.app.FragmentContainer.instantiate (FragmentContainer.java:53)
  at android.app.FragmentState.instantiate (FragmentState.java:77)
  at android.app.FragmentManagerImpl.restoreAllState (FragmentManager.java:2867)
  at android.app.FragmentController.restoreAllState (FragmentController.java:142)
  at android.app.Activity.onCreate (Activity.java:1585)
  at org.medicmobile.webapp.mobile.EmbeddedBrowserActivity.onCreate
  at android.app.Activity.performCreate (Activity.java:8000)
  at android.app.Activity.performCreate (Activity.java:7984)
  at android.app.Instrumentation.callActivityOnCreate (Instrumentation.java:1309)
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3464)
Caused by java.lang.NoSuchMethodException: t.v.<init> []
  at java.lang.Class.getConstructor0 (Class.java:2332)
  at java.lang.Class.getConstructor (Class.java:1728)
  at android.app.Fragment.instantiate (Fragment.java:543)
```

**Environment**
- Instance: ?
- Android Version: A range...
- App Version: recent

**Additional context**
More info here: https://play.google.com/console/developers/7987044497930745454/app/4973604959552855131/vitals/crashes?errorType=CRASH&appProcessState=Foreground&days=28

**Comment by sugat009** [comment]
Updates:
Putting this on pause status for the time being as this issue seems to be a bit more time-consuming than initially assumed. I'll pick up this issue once I have finished my current ongoing tasks in the app services team.
So far my attempts to recreate this issue have been unsuccessful. I've tried this on a wide range of devices, both Android Studio emulators and physical devices, in various Android versions.
Things to do when resuming this:
1. As Gareth suggested, instead of recreating the issue, we could start addressing the issue based on the error stack trace instead.
2. Reach out to project officers and survey if this issue has been reported by any CHWs.(I'll be doing this shortly.)
